### PR TITLE
Wildcards(Breakout Folder)

### PR DESCRIPTION
**Pull request by Luro223**
Added additional info about 2 characters from wildcards * and ?

### CODE REVIEW

**Review by Luro223 — COMMENTED** [review]
Additional changes

**Review by Luro223 — COMMENTED** [review]
Additional formatting

File: Content/BreakoutFolder.md [hunk 1]
```diff
@@ -11,12 +11,18 @@ Usage:
...
    BreakoutFolder=E:\
+   BreakoutFolder=C:\App\*
+   BreakoutFolder=C:\App?
 ```
 
 The first example specifies that any content inside the folder "C:\Downloads" will be launched unsandboxed.
 
 Entire drives can also be specified as shown in the second example.
 
+The third and fourth line shows basic characters from wildcard.
```
Comment (isaak654) [inline Content/BreakoutFolder.md]:
```suggestion
The third and fourth lines show basic characters from wildcards.
```

File: Content/BreakoutFolder.md [hunk 2]
```diff
@@ -11,12 +11,18 @@ Usage:
...
+`*` defines any subfolder beyond App folder(App\1, App\1\1 etc.).
```
Comment (isaak654) [inline Content/BreakoutFolder.md]:
```suggestion
`*` defines any subfolder beyond App folder (App\1, App\1\1 and etc.).
```

**Review by isaak654 — CHANGES_REQUESTED** [review]

File: Content/BreakoutFolder.md [hunk 3]
```diff
@@ -11,12 +11,18 @@ Usage:
...
+   BreakoutFolder=C:\App?
 ```
 
 The first example specifies that any content inside the folder "C:\Downloads" will be launched unsandboxed.
 
 Entire drives can also be specified as shown in the second example.
 
+The third and fourth lines show basic characters from wildcards.
+`*` defines any subfolder beyond App folder (App\1, App\1\1 and etc.).
+`?` defines single character from folder (App1, App2 .. App9) but not subfolders.
```
Comment (isaak654) [inline Content/BreakoutFolder.md]:
```suggestion
`?` defines a single character from folder (App1, App2 .. App9) but not subfolders.
```

**Review by isaak654 — CHANGES_REQUESTED** [review]

File: Content/BreakoutFolder.md [hunk 4]
```diff
@@ -11,12 +11,21 @@ Usage:
...
 
 The first example specifies that any content inside the folder "C:\Downloads" will be launched unsandboxed.
 
 Entire drives can also be specified as shown in the second example.
 
+The third and fourth lines show basic characters from wildcards.
+`*` defines any subfolder beyond App folder (App\1, App\1\1 and etc.).
+`?` defines a single character from folder (App1, App2 .. App9) but not subfolders.
+
+Also you can combine several wildcards to match specified folder name and subfolders
```
Comment (isaak654) [inline Content/BreakoutFolder.md]:
```suggestion
Also, you can combine several wildcards to match a specified folder name and subfolders
```
or

```suggestion
Also, you can combine several wildcards to match the specified folder name and subfolders
```

**Review by isaak654 — CHANGES_REQUESTED** [review]

File: Content/BreakoutFolder.md [hunk 5]
```diff
@@ -11,12 +11,21 @@ Usage:
...
 
 The first example specifies that any content inside the folder "C:\Downloads" will be launched unsandboxed.
 
 Entire drives can also be specified as shown in the second example.
 
+The third and fourth lines show basic characters from wildcards.
+`*` defines any subfolder beyond App folder (App\1, App\1\1 and etc.).
+`?` defines a single character from folder (App1, App2 .. App9) but not subfolders.
+
+Also, you can combine several wildcards to match the specified folder name and subfolders
```
Comment (isaak654) [inline Content/BreakoutFolder.md]:
```suggestion
Also, you can combine several wildcards to match the specified folder name and subfolders.
```

**Review by isaak654 — CHANGES_REQUESTED** [review]

**Review by isaak654 — APPROVED** [review]

**Review by isaak654 — APPROVED** [review]

**Pull request merged**